tests/openssl/tests_3.unsupported

```diff
@@ -38,8 +36,6 @@
 ../../3rdparty/openssl/openssl/test/siphash_internal_test.c
 # SM2 is disabled by OE.
 ../../3rdparty/openssl/openssl/test/sm2_internal_test.c
-# SM3 is disabled by OE.
-../../3rdparty/openssl/openssl/test/sm3_internal_test.c
```

Comment:
Same question for this test.

Reply:
I made a mistake in a previous PR adding this to unsupported, it's already in the supported test list